setup.py

```diff
@@ -29,6 +29,7 @@
     ],
     "docs": [
         "sphinx==4.3.1",
+        "sphinx_autodoctypehints==1.19.2",
```

Comment:
docs build fails with
```
ERROR: Could not find a version that satisfies the requirement sphinx_autodoctypehints==1.19.2 (from pyfstat[docs]) (from versions: none)
ERROR: No matching distribution found for sphinx_autodoctypehints==1.19.2
```

Reply:
now
```
ERROR: Cannot install PyFstat and pyfstat[docs]==1.17.0+26.3843e571.clean because these package versions have conflicting dependencies.

The conflict is caused by:
    pyfstat[docs] 1.17.0+26.3843e571.clean depends on sphinx==4.3.1
    sphinx-autodoc-typehints 1.19.2 depends on Sphinx>=5.1.1
```

I don't think there was a breaking change in Sphinx that we needed to pin to `4.3.1` specifically for, the pin is just there because of their suggestion to always pin to a known working configuration. So you can try to update the whole stack.